Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -130,58 +195,73 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this.element.appendChild(fontVariationPropertiesSection.element);
     }
 
-    _formatSizeValue(property)
+    // Private
+
+    get #fontPropertiesMap()
```

Comment:
When I said "private" I had initially meant "private" in the way we have typically handled class-private members (e.g. prefix them with `_`). I'm torn on this. On one hand, we should start using the actual private syntax, on the other hand we should probably resolve www.webkit.org/b/235859 before adopting private members, otherwise we begin to reduce the debug-ability of Web Inspector...

Reply:
Agreed. Private member syntax is very nice to use, but perhaps it can wait until proper Web Inspector support.
It looked off that only some members were private and others were just `_` "private". I'll keep it uniform.